.github/workflows/check_scala3_nightly.yml

```diff
@@ -11,7 +11,7 @@ jobs:
       - uses: actions/setup-java@v3
         with:
           distribution: "temurin"
-          java-version: ${{ matrix.java }}
+          java-version: "17"
```

Comment:
Whoops, this was a copy pasta error on my part 😬

Reply:
:smile: no - it was copy-pasted by me https://github.com/scalameta/metals/pull/4311/files

Reply:
Ahhhh even better than ha. I just assumed this was my mistake when I made the switch to setup-java.